packages/fields/src/types/OEmbed/views/Field.js

```diff
@@ -41,15 +41,15 @@ const PlaceholderPreview = ({ originalUrl, fieldPath }) => (
   />
 );
 
-export default class UrlField extends Component {
+export default class OEmbedField extends Component {
   onChange = event => {
     this.props.onChange({
       originalUrl: event.target.value,
     });
   };
 
   render() {
-    const { autoFocus, field, value, savedValue = {}, errors } = this.props;
+    const { autoFocus, field, value = {}, savedValue = {}, errors } = this.props;
```

Comment:
In an upcoming PR, the `value` passed through here could be `undefined`, so we default it to an empty object (the same as we did for `savedValue`).